ast/src/expression/unary.rs

```diff
@@ -16,10 +16,16 @@
 
 use super::*;
 
+/// The unary operator for an unary expression.
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub enum UnaryOperation {
+    /// The logical not operator, i.e., `!`.
```

Comment:
Could say logical negation. (Cf. other comments about disjunction and conjunction.)